app/controllers/api/v3/scenarios_controller.rb

```diff
@@ -480,8 +484,8 @@ def force_uncouple
       # scenario which caused the error.
       #
       # Returns the result of the block.
-      def wrap_with_sentry_context
-        ScenarioSentryContext.with_context(@scenario) { yield }
+      def wrap_with_sentry_context(&)
+        ScenarioSentryContext.with_context(@scenario, &)
```

Comment:
Haha you're thing went a bit overboard. It's good when things look more clean, but I think I'd prefer not to change too much as we will loose a clear git blame commit history on all these lines.

Reply:
Definitely right - I have made the changes!